README.md

```diff
@@ -123,6 +123,21 @@ made, and there is no point in running tests in all LLVM optimization modes.
 
 When the build succeeds, you can run the tests using [the examples below](#usage).
 
+## GitHub Actions
+
+The `era-compiler-tester` is integrated into the GitHub Actions workflows of the following projects:
+
+* [era-compiler-llvm](https://github.com/matter-labs/era-compiler-llvm)
+* [era-solidity](https://github.com/matter-labs/era-solidity/)
+
+To allow verification of the custom `era-compiler-tester` changes in the Pull Requests (PRs) of these repositories, two additional tags are supported:

```

Comment:
Let's say `testing custom FE and VM changes`.

Reply:
Done